tools/clang/test/CodeGenDXIL/literal/fmod_const_eval.hlsl

```diff
@@ -0,0 +1,43 @@
+// RUN: %dxc -T vs_6_0 %s -E main | FileCheck %s
+// RUN: not %dxc -T vs_6_0 %s -E main -DNO_FOLD 2>&1 | FileCheck %s --check-prefixes=NO_FOLD
+
+// The code path for this case asserts, but the assert does not indicate a
```

Comment:
I don't think this is accurate anymore either. You might just say that this code path produces an invalid overload